tests/cmake/README.md

```diff
@@ -5,6 +5,17 @@ CMake interoperability with the MPC-Generated OpenDDS codebase.
 
 ## Building and Running All the CMake Tests
 
+If you have a new enough CMake, this can be down with:
```

Comment:
```suggestion
If you have a new enough CMake, this can be done with:
```
```suggestion
If you have a new enough CMake, this can be down with:
```